lib/puppet/bindings.rb

```diff
@@ -129,7 +129,7 @@ def [](name)
     end
 
     def has_name?(name)
-      @named_bindings.has_key?
+      @named_bindings.has_key?(name)
```

Comment:
Still present in master at https://github.com/puppetlabs/puppet/blob/master/lib/puppet/bindings.rb#L132